guidelines/program-guidelines.md

```diff
@@ -0,0 +1,81 @@
+# Program Guidelines
+
+## Contributors
+
+- Ashley
+- Jorge
+- Ricky
+- Manish
+
+## Have a clear audience goal
+
+Before you review talks, have a clear idea of what your target audience is. This is not just “users of the technology the conference is about”, this is also includes subgroups by field or level of expertise. Beginner talks will bore advanced users, and advanced talks will not be useful for beginners. When you have multiple rooms, having multiple targets is more feasible; however this can be tricky to schedule, so be careful.
+
+## Introductions
+
+If you have an emcee, be sure to communicate with speakers about this well in advance. Ask how they’d like to be introduced — some may provide you with text, some may care less (in which case, prepare the text yourself and make sure the emcee has it well in advance). Make sure you know their pronouns, and have your emcee work with them to make sure their name will be pronounced correctly.
+
+Some speakers feel awkward about self-introduction, so having an emcee introduce them can be quite helpful. Others may have a talk that starts with an introduction that segues into the material, so they may wish for the emcee introduction to be shorter.
+
+Overall, be consistent with how folks are introduced.
+
+## Logisitics and Announcements
+
+A website or central document that describes the general practices and layout of the event is very valuable and can eliminate a lot of redundent questions.
+
+Try to keep all the information for the event in a central and logically organized manner. Keep in mind that weird things *will* happen so have a plan for emergency/urgent announcements (anyone have an HDMI cable?) and make sure people know where to look for these. Use the same medium (twitter/Slack/IRC/etc) as inconsistancy will lead to confusion or missed announcements. 
+
+Visit the venue at least once before the event, and prep the volunteers on the basics (especially if they are also coming in from out of town).
+
+## Things take time
+
+Although we “know” that things take time, we rarely schedule time for all the things, *beyond talks*,  that need to happen during an event.  Some of these things include:
+
+  - Applause (seriously)
+  - Plugging/Unplugging Laptops
+  - Speaker Introductions
+  - Announcements
```

Comment:
One thing we have at RustFest: Main sponsors get a 5min talk at the beginning. Might be worth to add as "Other" or something alike?